app/helpers/application_helper.rb

```diff
@@ -60,4 +60,12 @@ def link_to_change_answer(step)
   def link_to_git_site(text = "Get into Teaching", attributes = {})
     link_to text, (ENV["GIT_URL"].presence || "/url-not-set"), attributes
   end
+
+  def internal_referer
+    referer = request.referer
+    internal = referer.to_s.include?(root_url)
+    return nil unless internal
```

Comment:
is this short enough to just be `return nil unless referer.to_s.include?(root_url)`

Reply:
Yeh, agreed - its a cut and paste from `-app` so I'll leave it as is for cherry-picking for now - plan is to move it all to common code anyway